client/src/App.js

```diff
@@ -42,6 +44,10 @@ class App extends Component {
           <Route path="/contact" >
             <Contact />
           </Route>
+          <Route >
+            {/* Temporarily  in feedback we put rest. id*/}
```

Comment:
You can use commit message for this explanation. We should avoid comments in code

Reply:
Before PR I delete comment. Its explain to code for reviewers

Reply:
I think reviewers are smart enough ;)
And anyway they can ask question in PR .